CHANGELOG.md

```diff
@@ -2,23 +2,190 @@
 
 # 1.1.0
 
-- Bunch of new functions.
-- `PIntegral`.
-- Tracing utilities.
-- Documentation.
-- Lists.
-- `PLift` and `PContant`.
-- Rationals.
-- Many new optimisations.
-- `TermCont`.
-- `s :: S`.
-- Nix stuff.
-- GHC 9.2.1.
-- `do`-notation (!).
-- Benchmarks.
-- `letrec` and records.
-- More complete `Plutarch.Api.V1`.
-- Add generic deriving for `PIsDataRepr`
+- Repo changes (not visible on the user side)
+  - The Plutarch repo has moved to GHC 9.2.1 (GHC 8.10.7 should still be able to depend on Plutarch).
+
+    Relevant PR: [#86](https://github.com/Plutonomicon/plutarch/pull/86)
+  - Major nix updates.
+  - Benchmarks and perf diffing on PRs to keep track of performance regressions.
+
+    Worked on through:
+    * [#144](https://github.com/Plutonomicon/plutarch/pull/144)
+    * [#146](https://github.com/Plutonomicon/plutarch/pull/146)
+    * [#164](https://github.com/Plutonomicon/plutarch/pull/164)
+    * [#167](https://github.com/Plutonomicon/plutarch/pull/167)
+    * [#178](https://github.com/Plutonomicon/plutarch/pull/178)
+
+- Significantly improve `plam` type inference - works seamlessly now (!!!)
+
+  Worked on through:
+  * [#29](https://github.com/Plutonomicon/plutarch/pull/29)
+  * [#149](https://github.com/Plutonomicon/plutarch/pull/149)
+  * [#162](https://github.com/Plutonomicon/plutarch/pull/162)
+  * [#168](https://github.com/Plutonomicon/plutarch/pull/168)
+  * [#170](https://github.com/Plutonomicon/plutarch/pull/170)
+
+  Relevant issue: [#2](https://github.com/Plutonomicon/plutarch/issues/2)
+- Many, many optimizations on generated code.
+
+  Worked on through:
+  * [#34](https://github.com/Plutonomicon/plutarch/pull/34)
+  * [#37](https://github.com/Plutonomicon/plutarch/pull/37)
+  * [#42](https://github.com/Plutonomicon/plutarch/pull/42)
+  * [#44](https://github.com/Plutonomicon/plutarch/pull/44)
+- Add haddocks and examples to many functions
+
+  Started by: [#49](https://github.com/Plutonomicon/plutarch/pull/49)
+- Add many new utility functions, instances, and builtin function synonyms.
+
+  <details>
+  <summary> Breakdown of additions </summary>
+
+  - Boolean utilities:- `pnot`, `#&&`/`pand`/`pand'`, `#||`/`por`/`por'`
+
+    Module: `Plutarch.Bool`
+
+    Added in: [#30](https://github.com/Plutonomicon/plutarch/pull/30)
+  - ByteString utilities:- `pconsBS`, `psliceBS`, `plengthBS`, `pindexBS`
+
+    Module: `Plutarch.ByteString`
```

Comment:
Why have all of this `Module:` stuff?

Reply:
Easy for people to *see* the new changes and the type signatures for now. until we have some hosted haddocks to point to or similar. Also see: https://github.com/Plutonomicon/plutarch/pull/99#discussion_r786905746